exampleCourse/questions/demo/autograder/c/squareCriterion/tests/test.py

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/python3
+
+import cgrader
+
+
+class DemoGrader(cgrader.CGrader):
+    def tests(self):
+        self.test_compile_file(
+            "square.c",
+            "main",
+            add_c_file="/grade/tests/main.c",
+            # The following must be included if compiling a Criterion test
+            flags="-lcriterion",
+        )
+        self.run_criterion_suite("./main", use_iteration=True)
+
+
+g = DemoGrader()
```

Comment:
Should have these two lines under a name == main check.

Reply:
@eliotwrobson this is very much a WIP and not at all close to a finished product. There are tonnes of changes coming up, I don't think it's worth reviewing at this point.